qdrant-landing/content/articles/what-is-quantization.md

```diff
@@ -0,0 +1,552 @@
+---
+title: "What is Vector Quantization?"
+draft: false
+slug: what-is-vector-quantization 
+short_description:  What is Vector Quantization? Methods & Examples | Qdrant
+description: Learn what vector quantization is and explore how methods like Scalar, Product, and Binary Quantization work. Plus, find out how to choose the best method for your specific application.
+preview_dir: /articles_data/what-is-vector-quantization/preview
+weight: -210
+social_preview_image: /articles_data/what-is-vector-quantization/preview/social-preview.jpg
+date: 2024-09-16T09:29:33-03:00
+author: Sabrina Aquino 
+featured: true 
+tags: 
+  - vector-search
+  - vector-quantization
+  - binary quantization
+  - product quantization
+  - scalar quantization
+  - vector compression
+
+---
+
+> Vector quantization is a technique for reducing the size of high-dimensional data. Instead of storing the original vectors, the data is stored in a compressed format, which reduces memory usage while maintaining most of the essential information. This compression allows for more efficient storage and faster search operations, especially in large datasets.
+
+When working with high-dimensional vectors, such as embeddings from AI models like OpenAI, one single 1536-dimensional vector requires **6KB of memory**.
+
+<img src="/articles_data/what-is-vector-quantization/vector-size.png" alt="1536 dimentional vector size is 6KB" width="700">
+
+If your dataset has **millions of vectors**, the memory and processing demands are very significant.
```

Comment:
For sure! but since we're talking about millions of vectors, not just one million, I've adjusted some of the wording:

With 1 million vectors needing around 6GB of memory, as your dataset grows to multiple millions of vectors, the memory and processing demands increase significantly.